radixdlt-core/radixdlt/src/main/java/com/radixdlt/api/core/api.yaml

```diff
@@ -1017,11 +1044,6 @@ tags:
       the validators performing consensus. As a consequence of this, it is _not_ guaranteed to be increasing, and it
       may be affected by Byzantine nodes.
 
-      Not every transaction will have such a `RoundData` operation, for example, due to multiple transactions being
```

Comment:
Why was this removed?

Reply:
It's not really explained correctly as it implies *most* transactions will have a `RoundData` operation but no user transaction will have a `RoundData` operation. I'll refactor how this is worded